java/api/src/main/java/org/ray/api/function/PyActorMethod.java

```diff
@@ -0,0 +1,14 @@
+package org.ray.api.function;
+
+/**
+ * A class for calling actor method.
+ */
+public class PyActorMethod {
+  public String methodName;
+  public Class<?> returnType;
```

Comment:
This should be `Class<T>`. And the `call` method should also return `T`.

Reply:
`PyActorMethod<T>` too.

Reply:
I added generic type for PyActorMethod and PyRemoteFunction.